app/services/cfe_civil/submission_builder.rb

```diff
@@ -94,7 +95,7 @@ def process_response
       submission.cfe_result = @response.body
       if @save_result
         submission.results_obtained!
-        write_submission_history(@response, "POST")
+        build_submission_history(@response, "POST").save!
```

Comment:
nit: as the second param `http_method` has the same value as the default, do we need to specify it here?

Reply:
Probably not, this is a copy paste from the current CFE submission where it does vary.
In this brave new world the _only_ submission will be a `POST` so the entire thing could be refactored! 
🤔

Reply:
Have removed the superfluous call for now.  The *_submission_history pattern is in use in a number of places.  Extracting and centralising it may be more useful than slightly altering the pattern in one place :+1: